src/tcgwars/logic/impl/gen4/Platinum.groovy

```diff
@@ -2711,22 +2711,13 @@ public enum Platinum implements LogicCardInfo {
               int heads = 0;
               flipUntilTails {c++}
               if (c && opp.active.cards.filterByType(ENERGY)) {
-              opp.active.cards.filterByType(ENERGY).select(count:c,"Choose the energy cards to put in the Lost Zone").moveTo(opp.lostZone)
-            }
-          }
-        }
-          move "", {
-            text "Put this card onto your Active Dialga . Dialga LV. can use any attack, Poké-Power, or Poké-Body from its previous level."
-            energyCost ()
-            attackRequirement {}
-            onAttack {
-              damage 0
+                opp.active.cards.filterByType(ENERGY).select(count:c,"Choose the energy cards to put in the Lost Zone").moveTo(opp.lostZone)
+              }
```

Comment:
`c` --> `heads`

Just saved us a multi-year bug 😬 (see 2015-2018 energy requirement bug of Fates Collide cards)

Reply:
Nice catch, I've updated this